debiman-auxserver.service

```diff
@@ -1,5 +1,6 @@
 [Unit]
 Description=debiman auxiliary service endpoints
+Documentation=man:debiman-auxserver(1)
```

Comment:
We should probably move https://salsa.debian.org/go-team/packages/debiman/tree/master/debian/man into this repository in the same PR.

Reply:
Oh, yes!